docs/spl-docs/transaction-command.md

```diff
@@ -0,0 +1,207 @@
+# Transaction
+
+## Description
+
+The transaction command finds transactions based on events that meet various constraints. Transactions are made up of the raw text (the raw event) of each member, the time and date fields of the earliest member, as well as the union of all other fields of each member.
+
+Additionally, the `transaction` command adds two fields to the raw events, `duration` and `eventcount`. The values in the `duration` field show the difference between the timestamps for the `first` and `last` events in the transaction. The values in the `eventcount` field show the number of events in the transaction.
+
+In the output, the `events` in a transaction are grouped together as multiple values in the `Events` field. Each event in a transaction starts on a new line by default.
+
+
+## Syntax
+
+The required syntax is in **bold**.
+
+**transaction** \
+[\<field-list>] \
+[\<txn_definition-options>...]
+
+## Required Arguments
+
+None.
+
+## Optional Arguments
+
+### fied-list
+
+**Syntax**: `<field>...`
+**Description**: One or more field names. The events are grouped into transactions, based on the unique values in the fields. For example, suppose two fields are specified: `client_ip` and `host`. For each `client_ip` value, a separate transaction is returned for each unique `host` value for that `client_ip`.
+
+### txn_definition-options
+
+**Syntax**: `<startswith>` | `<endswith>` \
+**Description**: Specify the transaction definition options to define your transactions. You can use multiple options to define your transaction.
+
+### startswith
+
+**Syntax**: `startswith=<filter-string>` \
+**Description**: A search or eval filtering expression which if satisfied by an event marks the beginning of a new transaction.
+
+
+### endswith
+
+**Syntax**: `endswith=<filter-string>` \
+**Description**: A search or eval expression which, if satisfied by an event, marks the end of a transaction.
+
+
+
+## Filter String Options
+
+These options are used with the `startswith` and `endswith` arguments.
+
+#### \<filter-string>
+
+**Syntax**: `<search-expression> | (<quoted-search-expression>) | eval(<eval-expression>)` \
```

Comment:
okay then I should probably do like this right?

```
**Syntax**: `<search-expression>`, or `<quoted-search-expression>`, or `eval(<eval-expression>)`
```

I should mention `or` at every place where there is `|` otherwise users might assume in a different way!